content/python/concepts/strings/terms/strip/strip.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.strip()'
+Description: 'Eliminates spaces at the beginning and end of a string, and specified characters when passed as a parameter.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.strip()`** method removes any whitespace at the beginning and end of a string. It’s also possible to specify which characters will be removed; otherwise, all whitespaces will be removed. 
+
+## Syntax
+
+```pseudo
+string.strip(characters) 
+```
+
+Parameters are optional for this method.
```

Comment:
Generally we explain the parameters here. Briefly explain what `characters` takes/does